plugins/builds/update.js

```diff
@@ -68,62 +68,63 @@ module.exports = () => ({
                     build.status = desiredStatus;
 
                     // Only trigger next build on success
-                    return build.job.then(job => job.pipeline.then((pipeline) => {
-                        request.server.emit('build_status', {
-                            settings: job.permutations[0].settings,
-                            status: build.status,
-                            pipelineName: pipeline.scmRepo.name,
-                            jobName: job.name,
-                            buildId: build.id,
-                            buildLink: `${buildFactory.uiUri}/builds/${id}`
-                        });
-
-                        // Guard against triggering non-successful builds
-                        if (desiredStatus !== 'SUCCESS') {
-                            return null;
-                        }
-
-                        const workflow = pipeline.workflow;
+                    return build.update()
```

Comment:
The status is success for me. U need to change ur docker-compose file to pull in the api on this branch instead of pulling an image. Let me send u my docker-compose file on slack